components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Transcribe speech or music with Whisper'
-description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful standard library makes manipulating audio files and processing the results trivially simple."
+description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful Ballerina library makes manipulating audio files and processing the results trivially simple."
```

Comment:
It is not meaningful to say `Ballerina’s powerful Ballerina library` as Ballerina is repeating weirdly here.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.